README.md

```diff
@@ -114,38 +103,25 @@ activationRequest.run(new ResultCallback<Void>() {
     }
 });
 ```
- 
-Third, optional step is to fund the created account with KIN.  
-When working against KIN test network, the KIN Faucet Service can be used for this purpose:
-```java
-Request request = new Request.Builder()
-            .url("http://159.65.84.173:5000/fund?account=" + account.getPublicAddress() + "&amount=" + FUND_KIN_AMOUNT)
-            .get()
-            .build();
-okHttpClient.newCall(request)
-    .enqueue(new Callback() {
-        @Override
-        public void onFailure(@NonNull Call call, @NonNull IOException e) {
-            e.printStackTrace();
-        }
 
-        @Override
-        public void onResponse(@NonNull Call call, @NonNull Response response)
-            throws IOException {
-            if (response.code() == 200) {
-              //Account was funded with KIN succesfully
-            }
-        }
-    });
-```
+For more details see [Onboarding](onboarding.md), also take a look at Sample App [OnBoarding](https://github.com/kinecosystem/kin-core-android/blob/dev/sample/src/main/java/kin/core/sample/OnBoarding.java) class for a complete example.
```

Comment:
The sample app is super "weird", the onboarding there doesn't look like the code the documentation "suggests?

Reply:
Are you talking about this? https://kinecosystem.github.io/kin-core-docs/docs/onboarding.html